### PR TITLE
Add 8 v2v cases of converting esx guest to rhev

### PR DESCRIPTION
**Pull request by chloerh**

### CODE REVIEW

**Comment by chloerh** [comment]
test result:

> JOB ID     : 7114ddd439aa4f0593fb78a7ae0be65a547c7a89
> JOB LOG    : /root/avocado/job-results/job-2016-05-17T14.25-7114ddd/job.log
> TESTS      : 8
>  (1/8) type_specific.local.esx_vm.with_cdrom.esx_55.rhev: PASS (1279.01 s)
>  (2/8) type_specific.local.esx_vm.migrated_vm.esx_55.rhev: PASS (1753.07 s)
>  (3/8) type_specific.local.esx_vm.local_storage.esx_55.rhev: PASS (1310.62 s)
>  (4/8) type_specific.local.esx_vm.multiple_disks.esx_55.rhev: PASS (1333.51 s)
>  (5/8) type_specific.local.esx_vm.multiple_cpus.esx_55.rhev: PASS (1331.48 s)
>  (6/8) type_specific.local.esx_vm.special_name.esx_55.rhev: FAIL (8.19 s)
>  (7/8) type_specific.local.esx_vm.cloned_vm.esx_55.rhev: PASS (1229.96 s)
>  (8/8) type_specific.local.esx_vm.esx_template.esx_55.rhev: PASS (1251.10 s)
> RESULTS    : PASS 7 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0
> JOB HTML   : /root/avocado/job-results/job-2016-05-17T14.25-7114ddd/html/results.html
> TIME       : 9496.95 s

1 failed case please refer to [Bug 665883 - virt-v2v: Special character in vm name from vmware ESX not showing correct](https://bugzilla.redhat.com/show_bug.cgi?id=665883)

File: v2v/tests/src/esx_vm.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            'hostname': remote_host, 'hypervisor': 'esx', 'main_vm': vm_name,
+            'vpx_dc': vpx_dc, 'esx_ip': esx_ip,
+            'v2v_opts': '-v -x', 'input_mode': 'libvirt',
+            'storage': params.get('output_storage', 'default'),
+            'network': params.get('network'),
+            'bridge':  params.get('bridge'),
+            'target':  params.get('target')
+        }
+
+        os.environ['LIBGUESTFS_BACKEND'] = 'direct'
```
Comment (will-Do) [inline v2v/tests/src/esx_vm.py]: IIRC, this env var is for Xen VM convert

File: v2v/tests/cfg/esx_vm.cfg [hunk 2]
```diff
@@ -0,0 +1,90 @@
+- esx_vm:
```
Comment (will-Do) [inline v2v/tests/cfg/esx_vm.cfg]: It's better to rename the case name to make more readable

**Comment by chloerh** [comment]
@will-Do 
case name update, is the current name ok?

**Comment by will-Do** [comment]
OK, merge this pr

**Pull request merged**